logstage-elastic/src/main/scala/Logging.scala

```diff
@@ -17,8 +17,15 @@ trait Logging extends LoggerTags {
 
   implicit lazy val log: IzLogger = {
     val legacyNameNormalisation = sys.props.get("logger.izumi.legacyNameNormalisation").contains("true")
-
-    lazy val jsonPolicy = LogstageCirceElasticRenderingPolicy(loggerClass, legacyNameNormalisation)
+    // set this to 0 to turn it off
+    val truncateStringValues = sys.props.get("logger.izumi.truncateStringValuesTo").map(_.toInt)
+    val defaultTruncateStringValuesTo = 16_384 // 16KB
```

Comment:
or 16 * 1024, but I guess a comment is needed anyway

Reply:
changed to the multiplication but I left out the bytes suffix. as to be exact it's the length of the string we're referring too (not the length of it's bytes representation which might differ with unicode afaik)